plume-models/src/comments.rs

```diff
@@ -1,4 +1,4 @@
-use activitypub::{activity::Create, link, object::Note};
+use activitypub::{activity::{Create, Delete}, link, object::{Note, Tombstone}};
```

Comment:
Tombstone… sounds cool

Reply:
It's what you need to live behind when deleting things (as opposed to undoing)